src/app/@client/(content)/home/page.tsx

```diff
@@ -1,15 +1,35 @@
 import BlogShowcase from '@/components/Blogs/BlogShowcase';
 import React from 'react';
 import SimpleCTA from '@/components/CTA/SimpleCTA';
+import ClientScheduleAppointment from '@/components/Calendars/ClientScheduleAppointment';
+import { serverFetcher } from '@/lib/fetchers/serverFetcher';
+import { StrapiResponse } from '@/types/StrapiResponse';
+import { Availability } from '@/types/Availability';
+
+const fetchAvailabilities = async () => {
+    const today = new Date().toISOString().substring(0, 10);
+
+    return serverFetcher<StrapiResponse<Availability>>({
+        url: `/nutritionist-availabilities?populate[nutritionist][populate]&filters[nutritionist][id]=2&filters[date][$gte]=${today}`,
+        method: 'get',
+    });
+};
+
+export default async function Home() {
+    const availabilities = await fetchAvailabilities();
 
-export default function Home() {
     return (
         <>
             <SimpleCTA
                 title="Parece que ainda não tem nenhuma consulta marcada..."
                 description="Reserve agora a sua primeira consulta."
                 buttonText="Marcar consulta"
             />
+
+            <ClientScheduleAppointment
+                availableDates={availabilities.data.map((av: Availability) => new Date(av.attributes.date))}
+            />
```

Comment:
This is incorrect, you will lose track of what the availability the client has chosen if you just send the dates:

We need to maintain the ID and everything, so for that, please send the full availabilities as a prop and if you want to get the dates from them, do this inside the component instead